classes/MyTableAdmin.php

```diff
@@ -511,13 +549,14 @@ public function recordSave($messageSuccess = false, $messageError = false)
             }
         } else {
             Tools::addMessage('info', $this->translate('Nothing to save.'));
+            // todo ask CRS2  Method GodsDev\MyCMS\MyTableAdmin::recordSave() should return bool but returns int.
             return 0;
```

Comment:
@crs2 metoda by měla vrátit bool, ale zde vrací 0. Co s tím?

Reply:
Tady prosím pozměnit phpDoc u této metody (recordSave) na (něco jako):
/* @return bool|int - true = record saved sucessfully, false = error occured saving the record, 0 = no records to save
*/

Reply:
Prima. To todo jsem přidal k metodě. V další fázi udělám. Slávek